sphinx-docs/Exfiltration.md

```diff
@@ -0,0 +1,28 @@
+Exfiltration
+====================
+
+After completing an operation a user may want to review the data retreived from the target system. This data is automatically stored on the CALDERA server in a directory specified in [/conf/default.yml](Server-configuration.html#the-existing-default-yml).
+
+## Accessing Exfiltrated Files
+
+Some abilities will return files from the agent to the CALDERA server. This can also be done manually with 
+```yaml
```

Comment:
"shell" or nothing will be more appropriate than "yaml" here (option this determines syntax highlighting)